tests/test_encodings.py

```diff
@@ -420,10 +421,43 @@ def test_mds_float64(self, decoded: float, encoded: bytes):
         assert isinstance(dec, np.floating)
         assert dec == decoded
 
+    @pytest.mark.parametrize(('decoded', 'encoded'), [(42, b'42')])
```

Comment:
What would happen when it is a negative integer ? Can you please also add the test case for the same?